packages/attestation-service/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@celo/contractkit": "1.5.3-dev",
+    "@celo/identity": "1.5.3-dev",
     "@celo/keystores": "1.5.3-dev",
```

Comment:
I think adding the `@celo/phone-utils` dep here and in the Dockerfile should fix the failing build